src/syscheckd/src/db/tests/db/ComponentTest/registryInterface/registryTest.h

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
This file could be deleted because there are a fixture defined on dbTest.h which has the same behavior

Reply:
done